fe/fe-core/src/main/java/org/apache/doris/load/DeleteHandler.java

```diff
@@ -709,6 +745,51 @@ public void write(DataOutput out) throws IOException {
 
     public static DeleteHandler read(DataInput in) throws IOException {
         String json = Text.readString(in);
-        return GsonUtils.GSON.fromJson(json, DeleteHandler.class);
+        DeleteHandler deleteHandler = GsonUtils.GSON.fromJson(json, DeleteHandler.class);
+        deleteHandler.removeOldDeleteInfos();
+        return deleteHandler;
+    }
+
+    /**
+     * FE Master timing, or when any FE replay meta triggers DeleteInfo cleanup
+     */
+    public void removeOldDeleteInfos() {
+        Timestamp timestamp = new Timestamp();
+        removeOldDeleteInfos(timestamp);
+        Catalog.getCurrentCatalog().getEditLog().logRemoveOldDeleteInfos(timestamp);
```

Comment:
I think you can add a judgment here that when Fe is loading image, it will not execute this line.

Reply:
It has been fixed, please review it again, thank you very much : )

When the master or follower FE loads the image when it starts, old deleteInfos will not clean up and generate editlog.
FE Master timing or checkpoint loading image, trigger DeleteInfo cleanup.